larray_editor/arraymodel.py

```diff
@@ -108,8 +109,8 @@ def data(self, index, role=Qt.DisplayRole):
     def reset(self):
         self.beginResetModel()
         self.endResetModel()
-        if __debug__:
-            print("model reset", self.__class__)
+        logger.debug("model {} has been reset after call of {} from file {} line {}".format(self.__class__,
```

Comment:
Using the logger for this, is a great idea. This is meant for that. However I wonder if inspecting the stack frame to build the message is not too expensive and if we shouldn't check beforehand if the logger is enabled before constructing the message, with something like:

https://docs.python.org/3/library/logging.html#logging.Logger.isEnabledFor